EIPS/eip-2612.md

```diff
@@ -1,11 +1,10 @@
 ---
 eip: 2612
-title: "EIP-20 Permit Extension: Signed Approvals"
+title: "Permit Extension for EIP-20: Signed Approval"
```

Comment:
```suggestion
title: Permit Extension for EIP-20: Signed Approval
```

Reply:
This is not valid YAML. The quotes are needed.

![image](https://user-images.githubusercontent.com/481465/196059718-a1be7ab0-f56e-4765-84e2-c8fd9f8d33d8.png)

Reply:
i see. @SamWilsn and @Pandapip1 can we check if preemble is being linted as YAML?

Reply:
@frangio  But before that, would you consider the option to just drop column so it will automatically qualify without being blocked by EIPW's issue (or bug, I suspect)

Reply:
Do you mean dropping the colon? I guess that works.